tests/unit/work.test.ts

```diff
@@ -155,8 +156,8 @@ describe("Work Context", () => {
     });
 
     it("should catch error while executing batch as stream with invalid command", async () => {
-      const activity = await Activity.create("test_agreement_id", yagnaApi);
-      const worker = async (ctx: WorkContext) => ctx.beginBatch().run("invalid_shell_command").endStream();
+      const activity = await Activity.create(agreement, yagnaApi);
+      const worker = async (ctx) => ctx.beginBatch().run("invalid_shell_command").endStream();
```

Comment:
why remove `: WorkContext` from all workers in this file? If we make changes to `WorkContext` TS will not pick up that these files break

Reply:
it was probably a mistake when merging the beta